app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  after_save :updates_likes_counter
+
+  private
+
+  def updates_likes_counter
+    post.update(likes_counter: post.likes_counter + 1)
+  end
```

Comment:
[OPTIONAL] Great job implementing the functionalities 👍 . I think for scaling purposes, you can consider writing your method this way 👇  for the post and other ones..
 ```
 def updates_likes_counter
    post.increment!(:likes_counter)
  end
```